### PR TITLE
Implement cumulative aggregation

### PR DESCRIPTION
**Pull request by phofl**

### CODE REVIEW

File: dask_expr/_cumulative.py [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+def _take_last(a, skipna=True):
+    if skipna:
+        a = a.bfill()
+    return a.tail(n=1).squeeze()
+
+
+class TakeLast(Blockwise):
+    _parameters = ["frame", "skipna"]
+    operation = staticmethod(_take_last)
+    _projection_passthrough = True
```
Comment (crusaderky) [inline dask_expr/_cumulative.py:62]:
```suggestion
class TakeLast(Blockwise):
    _parameters = ["frame", "skipna"]
    _projection_passthrough = True
    
    @staticmethod
    def operation(a, skipna=True):
        if skipna:
            a = a.bfill()
        return a.tail(n=1).squeeze()
```

File: dask_expr/tests/test_collection.py [hunk 2]
```diff
@@ -242,6 +242,15 @@ def test_std_kwargs(axis, skipna, ddof):
...
 
+@pytest.mark.parametrize("func", ["cumsum", "cumprod", "cummin", "cummax"])
+def test_cumulative_methods(df, pdf, func):
+    assert_eq(getattr(df, func)(), getattr(pdf, func)(), check_dtype=False)
+    assert_eq(getattr(df.x, func)(), getattr(pdf.x, func)())
+
+    q = getattr(df, func)()["x"]
+    assert q.simplify()._name == getattr(df.x, func)()
+
+
```
Comment (crusaderky) [inline dask_expr/tests/test_collection.py:253]: Could you add a test for skipna=False (running on data that does include NA's)?

**Review by crusaderky — CHANGES_REQUESTED** [review]

File: dask_expr/tests/test_collection.py (hunk 2 above)
Comment (phofl) [reply dask_expr/tests/test_collection.py:262]: added

**Comment by phofl** [comment]
dask broke the ci

**Pull request merged**